### ISSUE TITLE
Sourcegenerator code produces warnings

### ISSUE THREAD

**Issue by espenrl** [CommunityToolkit/Maui#619]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The auto generated code produces warnings. In this case it breaks the build because the project is setup with `<AnalysisMode>All</AnalysisMode>` and `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`.

```
1>------ Build started: Project: FeltApp, Configuration: Debug Any CPU ------
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(12,29,12,60): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_Editor
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(12,29,12,64): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_DatePicker
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(12,29,12,60): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_Button
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(12,29,12,59): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_Entry
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(12,29,12,59): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_Label
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(12,29,12,60): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_Picker
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(12,29,12,65): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_RadioButton
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(12,29,12,63): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_SearchBar
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(12,29,12,64): error CA1707: Remove the underscores from type name Microsoft.Maui.Controls.ColorAnimationExtensions_TimePicker
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\SearchBarTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\DatePickerTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\LabelTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\ButtonTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\RadioButtonTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EditorTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\PickerTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\EntryTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(35,88,35,136): error CA2000: Call System.IDisposable.Dispose on object created by 'GetGreenTransformAnimation(element, color.Green)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(35,146,35,192): error CA2000: Call System.IDisposable.Dispose on object created by 'GetBlueTransformAnimation(element, color.Blue)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(35,34,35,78): error CA2000: Call System.IDisposable.Dispose on object created by 'GetRedTransformAnimation(element, color.Red)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(35,202,35,250): error CA2000: Call System.IDisposable.Dispose on object created by 'GetAlphaTransformAnimation(element, color.Alpha)' before all references to it are out of scope
1>C:\SourceCode\mdir\FeltApp\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\TimePickerTextColorTo.g.shared.cs(35,13,35,254): error CA2000: Call System.IDisposable.Dispose on object created by 'new Animation{{0, 1, GetRedTransformAnimation(element, color.Red)}, {0, 1, GetGreenTransformAnimation(element, color.Green)}, {0, 1, GetBlueTransformAnimation(element, color.Blue)}, {0, 1, GetAlphaTransformAnimation(element, color.Alpha)}, }' before all references to it are out of scope
```

### Expected Behavior

No warnings and also a `<auto-generated>` header should be included.

This is an example of a  `<auto-generated>` header from `Microsoft.Maui.Controls.SourceGen`.
``` CSharp

//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a .NET MAUI source generator.
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

[assembly: global::Microsoft.Maui.Controls.Xaml.XamlResourceId("Test.App.xaml", "App.xaml", typeof(global::Test.App))]
namespace Mdir
{
	[global::Microsoft.Maui.Controls.Xaml.XamlFilePath("App.xaml")]
	public partial class App : global::Microsoft.Maui.Controls.Application
	{
		[global::System.CodeDom.Compiler.GeneratedCode("Microsoft.Maui.Controls.SourceGen", "1.0.0.0")]
#if NET5_0_OR_GREATER
#endif
		private void InitializeComponent()
		{
			global::Microsoft.Maui.Controls.Xaml.Extensions.LoadFromXaml(this, typeof(App));
		}
	}
}

```

### Steps To Reproduce

``` xml
<AnalysisMode>All</AnalysisMode>
<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

### Link to public reproduction project repository

n/a

### Environment

```markdown
- .NET MAUI CommunityToolkit: 1.2.0
- OS: Windows 11
- .NET MAUI: 6.0
```

**Comment by marcmognol** [comment]
Hi, could you please check that PR #408 fixes your issue.
This PR which will be released soon fix the auto-generated flag.

**Comment by espenrl** [comment]
Yes, I could check it if I hadn't left for vacation today. I can do it in two weeks time.

I would add this to the sample csproj.

``` xml
<AnalysisMode>All</AnalysisMode>
<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

**Comment by maxkoshevoi** [comment]
Hi, I use latest (1.3.0) version of the toolkit, and warnings still appear:
https://github.com/maxkoshevoi/NureTimetable/actions/runs/3139381970/jobs/5099738174

**Comment by espenrl** [comment]
As far as I can see it is resolved in v1.3.0

**Comment by maxkoshevoi** [comment]
@espenrl so it is resolved for you? Intersecting, I'm still seeing the issue
